src/store/modules/RiskScoringModule.js

```diff
@@ -0,0 +1,15 @@
+import { VuexModule, Module, Action } from 'vuex-class-modules';
+import riskScoringService from '@/service/riskScoring';
+
+@Module({ generateMutationSetters: true })
+class RiskScoringModule extends VuexModule {
+  /**
+   * @return {Promise<void>}
+   */
+  @Action
+  async sendFingerprint() {
+    await riskScoringService.sendFingerprint();
```

Comment:
What about error handling?

Reply:
for now errors just suppressing
we should not show any notices or make some extra actions I guess

Reply:
yes, it should be in silent mode